lib/engines/content_object_store/config/routes.rb

```diff
@@ -4,7 +4,9 @@
 
     namespace :content_block, path: "content-block" do
       resources :documents, only: %i[index show], path_names: { new: "(:block_type)/new" }
-      resources :editions, only: %i[new create edit update], path_names: { new: "(:block_type)/new" }
+      resources :editions, only: %i[new create edit update], path_names: { new: "(:block_type)/new" } do
+        get :confirm, on: :member
```

Comment:
maybe a better name is `confirm_publish` ?

Reply:
Maybe even `review`, as that's what we're doing here

Reply:
ah yeah that's a good one too.... you don't think it needs to have `publish` in there?

Reply:
Nah, I think `review` is clear enough

Reply:
done!